src/api/include/migraphx/migraphx.hpp

```diff
@@ -1267,8 +1351,10 @@ struct experimental_custom_op : interface_base<MIGRAPHX_HANDLE_BASE(experimental
                              obj,
                              get_type_name(obj).c_str(),
                              obj.name().c_str());
-        MIGRAPHX_INTERFACE_LIFT(T, experimental_custom_op, compute_shape);
-        MIGRAPHX_INTERFACE_LIFT(T, experimental_custom_op, compute);
+        MIGRAPHX_INTERFACE_LIFT(T, experimental_custom_op, compute_shape, out_params<1>{});
```

Comment:
The out params should be passed as an integer: `MIGRAPHX_INTERFACE_LIFT(T, experimental_custom_op, compute_shape, 1)` then in the macro you can make it `out_params<1>{}`.

Reply:
Actually I would prefer to put this at the front though: `MIGRAPHX_INTERFACE_LIFT(1, T, experimental_custom_op, compute)`

Reply:
Done